src/ActionUnit.cpp

```diff
@@ -167,11 +167,11 @@ void ActionUnit::reParentAction(int childID, int oldParentID, int newParentID, i
             if (pChild->mLocation == 3) {
                 mpHost->mpConsole->mpRightToolBar->layout()->removeWidget(pChild->mpEasyButtonBar);
             }
+        }
```

Comment:
The original logic in this bit looked off to me - in that it would only pull a `TToolBar` from the application's dock areas (for `QDockWidget` derived classes) if there was ALSo a `TEasyButtonBar` associated with the `TAction` that `pChild` is - which there is not normally (well, now anyhow)...